prompttools/playground/data_loader.py

```diff
@@ -35,8 +35,10 @@ def load_data(
     presence_penalty=0.0,
     api_key=None,
 ):
+    data_loader_response = {'error': []}
```

Comment:
For error handling, why not just raise an error? Streamlit will show that to the user